gaia-sdk-java/gaia-sdk-api/src/main/kotlin/gaia/sdk/api/skill/SkillRef.kt

```diff
@@ -19,6 +20,25 @@ class SkillRef(private val spec: ISkillSpec, private val client: GaiaStreamClien
         return Flowable.fromPublisher(client.post(mapOf("uri" to spec.toUri(), "payload" to payload), SkillEvaluation::class.java, "/skill/evaluate"))
     }
 
+    /**
+     * Sends the given payload (a map satisfying contract) to this skill provision.
+     *
+     * The response is wrapped in a [SkillEvaluation].
+     */
+    fun evaluate(payload: Map<String, Any>, timeout: Duration): Publisher<SkillEvaluation> {
```

Comment:
Warum wird das nicht wie in Python mit default Parameter gemacht?

Reply:
Weil der default parameter in python nur deswegen ist, weil python method overloading nur mit default parameter kann.

Der default Wert für das Timeout soll nämlich eigentlich am Server definiert werden und nicht im Client. Deswegen ist das in Python nur so weil es nicht anders gegangen ist. Aber ich werde das noch anpassen, dass zumindest in Python kein expliziter Default Wert drin steht sondern None.